docs/adr/005_observability.md

```diff
@@ -74,24 +74,19 @@ In addition to this, we will support Application Insights metrics on an opt-in b
+| ReceiveWindowHits          | Number of times we hit the different receive windows.        | LNS    | LoRaWan   | Gateway Id, (estimated) Receive Window |
+| ReceiveWindowMisses        | Number of missed on downstream windows                       | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceCacheHit             | Number of device cache hit                                   | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceLoadRequests         | Number of device load requests                               | LNS    | LoRaWan   | Gateway Id                 |
 | JoinRequests               | Number of join requests                                      | LNS    | LoRaWan   | Gateway Id                 |
 | StationConnectivityLost    | Connection to LBS lost                                       | LNS    | LoRaWan   | Gateway Id                 |
 | ActiveStationConnections   | Active connections to stations                               | LNS    | LoRaWan   | Gateway Id                 |
 | D2CMessagesReceived        | Number of messages received from device                      | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessageDeliveryLatency  | Time from when we received the message from the concentrator until we are done processing it | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessagesDelivered       | Number of messages sent to upstream                          | LNS    | LoRaWan   | Gateway Id, To             |
-| D2CMessagesError           | Number of errors in sending messages to upstream             | LNS    | LoRaWan   | Gateway Id, To            |
-| D2CMessagesProcessingError | Number of errors processing (decoding, decrypting) messages  | LNS    | LoRaWan   | Gateway Id, To            |
+| D2CMessageDeliveryLatency  | Time from when we dispatched the message sent from the concentrator until we are done processing it | LNS | LoRaWan | Gateway Id |
 | D2CMessageSize             | Message size in bytes received from device                   | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessageSizeUpstream     | Message size in bytes sent upstream                          | LNS    | LoRaWan   | Gateway Id, To            |
 | C2DMessagesAbandoned       | Number of C2D messages abandoned                             | LNS    | LoRaWan   | Gateway Id                 |
-| C2DMessagesDelivered       | Number of messages sent downstream                           | LNS    | LoRaWan   | Gateway Id, To             |
-| C2DMessagesError           | Number of errors in sending messages to downstream           | LNS    | LoRaWan   | Gateway Id, To             |
-| C2DMessageSizeDownstream   | Message size in bytes sent downstream                        | LNS    | LoRaWan   | Gateway Id, To             |
+| C2DMessageSizeDownstream   | Message size in bytes sent downstream                        | LNS    | LoRaWan   | Gateway Id                 |
```

Comment:
What's the reason for this? Isn't it enough, if we know they are too long? Not sure what we will derive from this number and how we act on it?

Reply:
Removed with 213b3ce25d47e9330be9c718e390697c0c77dc3c